internal/storage/storageutil/object_attrs.go

```diff
@@ -9,22 +9,26 @@ import (
 	storagev1 "google.golang.org/api/storage/v1"
 )
 
+func convertACLRuleToObjectAccessControl(element storage.ACLRule) *storagev1.ObjectAccessControl {
+	return &storagev1.ObjectAccessControl{
+		Entity:   string(element.Entity),
+		EntityId: element.EntityID,
+		Role:     string(element.Role),
+		Domain:   element.Domain,
+		Email:    element.Email,
+		ProjectTeam: &storagev1.ObjectAccessControlProjectTeam{
+			ProjectNumber: element.ProjectTeam.ProjectNumber,
+			Team:          element.ProjectTeam.Team,
+		},
+	}
+}
+
 func ObjectAttrsToBucketObject(attrs *storage.ObjectAttrs) *gcs.Object {
 	// Converting []ACLRule returned by the Go Client into []*storagev1.ObjectAccessControl which complies with GCSFuse type.
 	var acl []*storagev1.ObjectAccessControl
+
```

Comment:
Remove this line.

Also, can you please re-write the above comment :-
// gcs.Object accepts []*storagev1.ObjectAccessControl instead of []ACLRule, hence need to convert.
Or similar.

Reply:
fixed